.template/addons/svgeez/template.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+apply '.template/addons/svgeez/Gemfile.rb'
```

Comment:
is this in the web project only @mosharaf13, for API variant, I think we don't need this gem